src/main/io/vtx_smartaudio_cms.h

```diff
@@ -0,0 +1 @@
+extern CMS_Menu cmsx_menuVtxSmartAudio;
```

Comment:
For the smartaudio, corresponding CMS contents reside in io/vtx_smartaudio.c.

I first thought it would be nice to keep them together with objects and functions the are related with. However, I'm not so sure now, thinking about the possible consolidation of vtx related APIs, associated codes, operational model and of course, UIs. There is also a couple of new FC controllable vtx that we are expecting to join this game. So, my plan is to keep it as is for 3.1, and restructure it as a part of the vtx system reorganization later (3.2 or 3.3).